core/src/main/java/com/datastax/oss/driver/internal/core/resolver/defaultResolver/DefaultResolver.java

```diff
@@ -0,0 +1,46 @@
+package com.datastax.oss.driver.internal.core.resolver.defaultResolver;
```

Comment:
Do we need to have a separate folder for it? Can we put it into `java/com/datastax/oss/driver/internal/core/resolver` ?

Reply:
We don't need to, but I thought it would make sense to have each resolver implementation in another subdirectory, just like mock one.
Removed the subdirectory.

Reply:
Sorry, you are correct, let's keep it there